readme.md

```diff
@@ -94,6 +94,8 @@ programming languages with appropriate bindings for Ruby.
 
 ### Frameworks
 
+- [LangChain.rb](https://github.com/andreibondarev/langchainrb) -
+  Build ML/AI-supercharged applications with Ruby's LangChain
```

Comment:
Please add the trailing dot.
Any specific reason why this post should stay at the beginning of the list, @andreibondarev ?

Reply:
Nope, but I didn't see any specific ordering in place.